docs/jailer.md

```diff
@@ -46,8 +46,10 @@ jailer --id <id> \
   jailer will use this to join the associated network namespace.
 - When present, the `--daemonize` flag causes the jailer to cal `setsid()` and
   redirect all three standard I/O file descriptors to `/dev/null`.
-- When present, the `--new-pid-ns` flag causes the jailer to `fork()` and then
-  exec the provided binary into a new PID namespace. As a result, the jailer and
+- When present, the `--new-pid-ns` flag causes the jailer to spawn the provided
```

Comment:
There is also a reference to `new-pid-ns` here: https://github.com/firecracker-microvm/firecracker/blob/a39fabbe1ab39af41a154142858ce9e981414fa5/docs/jailer.md#jailer-operation. Please update that too.